### PR TITLE
RCPS-156 User Management System: Create Documentation for Student Model

### PR DESCRIPTION
**Pull request by kylmrcdr123**
This is the documentation for the student model of user management system

### CODE REVIEW

File: src/main/java/com/user/management/appl/model/student/Student.java [hunk 1]
```diff
@@ -1,106 +1,213 @@
...
     private String email;
+
+    /** The address of the Student. */
     private String address;
+
+    /** The contact number of the Student. */
     private String contactNumber;
 
     /**
      * This is the Student getter and setter.
```
Comment (RoCS2024) [inline src/main/java/com/user/management/appl/model/student/Student.java:38]: what is this?

File: src/main/java/com/user/management/appl/model/student/Student.java [hunk 2]
```diff
@@ -1,106 +1,213 @@
...
- * This is the Student Model.
+ * POJO class for Employee. This class represents the User Management Employee.
  * */
 public class Student {
-    /**
-     * This is the Student Field.
-     * */
+
+    /** A unique id of the Student. */
     private String studentId;
```
Comment (RoCS2024) [inline src/main/java/com/user/management/appl/model/student/Student.java:8]: remove documentation on private properties

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
see comments

File: src/main/java/com/user/management/appl/model/student/Student.java (hunk 2 above)
Comment (kylmrcdr123) [reply src/main/java/com/user/management/appl/model/student/Student.java:8]: noted po

File: src/main/java/com/user/management/appl/model/student/Student.java [hunk 3]
```diff
@@ -1,106 +1,204 @@
...
+
     private String email;
+
     private String address;
+
     private String contactNumber;
 
+
     /**
      * This is the Student getter and setter.
```
Comment (RoCS2024) [inline src/main/java/com/user/management/appl/model/student/Student.java:29]: still did not remove

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
see one last comment

**Review by RoCS2024 — APPROVED** [review]
all good

**Pull request merged**